examples/usb_serial.rs

```diff
@@ -0,0 +1,97 @@
+//! CDC-ACM serial port example using polling in a busy loop.
+//! Target board: any STM32F7 with a OTG FS peripheral and a 25MHz HSE generator
+#![no_std]
+#![no_main]
+
+use panic_semihosting as _;
+
+use cortex_m_rt::entry;
+#[cfg(feature = "usb_fs")]
+use stm32f7xx_hal::otg_fs::{UsbBus, USB};
+#[cfg(feature = "usb_hs")]
+use stm32f7xx_hal::otg_hs::{UsbBus, USB};
```

Comment:
Do you also plan to add `usb_hs` support in a future PR? Otherwise this should probably be removed.

Reply:
Yes, I'm working on this part and I'm planning to add HS support in the future.